tests/time_tests/src/timetests_helper/cli.h

```diff
@@ -26,17 +26,9 @@ static const char target_device_message[] =
     "plugin. "
     "The application looks for a suitable plugin for the specified device.";
 
-/// @brief message for vpu argument
-static const char performance_hint_message[] =
-    "Not required. Enables performance hint 'LATENCY' for specified device.";
-
 /// @brief message for cache argument
-static const char cpu_cache_message[] =
-    "Not required. Use this key to run timetests with CPU models caching.";
-
-/// @brief message for vpu argument
-static const char vpu_compiler_message[] =
-    "Not required. Use this key to run timetests with 'MLIR' or 'MCM' VPUX compiler type.";
+static const char model_cache_message[] =
+    "Not required. Use this key to run timetests with models caching.";
```

Comment:
please, extend description with info that executable should be run twice - the second run will use cache prepared from first one

Reply:
done